docs/infrastructure/requesting-infrastructure.md

```diff
@@ -0,0 +1,59 @@
+# Requesting infrastructure
+
+This page details the process for requesting new infrastructure. This aims to be
+as lightweight as possible, while capturing all the information that's needed.
+
+Unless otherwise noted, the way to request things is to contact the
+Infrastructure Team by starting a new conversation thread in the
+[`#infrastructure` channel][slack-infra] in Slack. Please ping
+`@infrastructure-committee` to ensure that your message is seen.
+
+In general you can expect a response within a couple of days (often much
+sooner), though note that the team may not always be able to act on your request
+for some time.
+
+Requests for new infrastructure must be made by (or confirmed by) members of a
+Team Committee.
```

Comment:
Nit
```suggestion
Team's Committee.
```

Reply:
A "Team Committee" is a thing too, so I think this is fine as is.